[2025-10-11T06:32:56.835+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-11T06:32:56.887+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_flow.check_create_stg_tables scheduled__1999-12-19T00:00:00+00:00 [queued]>
[2025-10-11T06:32:56.900+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_flow.check_create_stg_tables scheduled__1999-12-19T00:00:00+00:00 [queued]>
[2025-10-11T06:32:56.903+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-11T06:32:56.925+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): check_create_stg_tables> on 1999-12-19 00:00:00+00:00
[2025-10-11T06:32:56.938+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=13581) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-11T06:32:56.940+0000] {standard_task_runner.py:64} INFO - Started process 13585 to run task
[2025-10-11T06:32:56.939+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipeline_flow', 'check_create_stg_tables', 'scheduled__1999-12-19T00:00:00+00:00', '--job-id', '1781', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmpl7cc_gy6']
[2025-10-11T06:32:56.943+0000] {standard_task_runner.py:91} INFO - Job 1781: Subtask check_create_stg_tables
[2025-10-11T06:32:57.029+0000] {task_command.py:426} INFO - Running <TaskInstance: pipeline_flow.check_create_stg_tables scheduled__1999-12-19T00:00:00+00:00 [running]> on host 321597a50810
[2025-10-11T06:32:57.183+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='oluwaseyiogundimu10@gmail.com' AIRFLOW_CTX_DAG_OWNER='seyi' AIRFLOW_CTX_DAG_ID='pipeline_flow' AIRFLOW_CTX_TASK_ID='check_create_stg_tables' AIRFLOW_CTX_EXECUTION_DATE='1999-12-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__1999-12-19T00:00:00+00:00'
[2025-10-11T06:32:57.185+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-11T06:32:57.204+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:32:57.303+0000] {connection.py:43} INFO - Table 'stg_users' in Postgres.
[2025-10-11T06:32:57.305+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:32:57.402+0000] {connection.py:43} INFO - Table 'users' in Postgres.
[2025-10-11T06:32:57.404+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:32:57.499+0000] {connection.py:43} INFO - Table 'stg_movies' in Postgres.
[2025-10-11T06:32:57.501+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:32:57.596+0000] {connection.py:43} INFO - Table 'movies' in Postgres.
[2025-10-11T06:32:57.598+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:32:57.695+0000] {connection.py:43} INFO - Table 'stg_ratings' in Postgres.
[2025-10-11T06:32:57.697+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:32:57.803+0000] {connection.py:43} INFO - Table 'ratings' in Postgres.
[2025-10-11T06:32:57.804+0000] {load.py:32} INFO - Tables exist.
[2025-10-11T06:32:57.806+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-11T06:32:57.807+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-11T06:32:57.827+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipeline_flow, task_id=check_create_stg_tables, run_id=scheduled__1999-12-19T00:00:00+00:00, execution_date=19991219T000000, start_date=20251011T063256, end_date=20251011T063257
[2025-10-11T06:32:57.880+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-10-11T06:32:57.899+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
